code/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
```

Comment:
Please create for files some nice folder structure.
On the top I created folder code ... there should be all code files but you should still create some structure ...
code/html/
code/js/
code/css ....

Reply:
folder structure applied, nice and neat